### PR TITLE
Fix dodgy dependency list

### PR DESCRIPTION
**Pull request by keirog**
These packages are all required in production, so shouldn’t be in the
devDependencies, which are getting pruned in Heroku.

Context: I recently added some logging to this app so we can audit usage (with the goal of switching it off soon) but new builds are failing because of this dependency list.

 🐿 v2.12.0

### CODE REVIEW

**Comment by j-mes** [comment]
:eyes: _dodgy!_

**Comment by j-mes** [comment]
Mind you, I wonder what can be done with the security checks with snyk

File: package.json [hunk 1]
```diff
@@ -6,15 +6,29 @@
...
+    "babel-preset-es2015": "^6.16.0",
+    "babel-preset-es2017": "^6.16.0",
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-preset-stage-1": "^6.16.0",
     "babel-preset-stage-3": "^6.17.0",
     "body-parser": "^1.15.2",
+    "bootstrap": "3.3.6",
     "bootstrap-webpack": "0.0.5",
+    "bower": "^1.7.9",
```
Comment (AbdullahAli) [inline package.json]: 😱

**Comment by AbdullahAli** [comment]
I can't see the snyk page anymore - it removed me from the team 

🤔

**Comment by keirog** [comment]
We wanna turn this app off soon so I’ve expedited this PR (i.e. ignoring the snyk message 😬 )

**Comment by keirog** [comment]
@AbdullahAli Are you signing in via SSO and your FT email address?

**Pull request merged**